CHANGELOG.next.asciidoc

```diff
@@ -42,7 +42,10 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 - Removed deprecated Sophos UTM from Beats. Use the https://docs.elastic.co/integrations/sophos[Sophos] Elastic integration instead. {pull}38037[38037]
 - Introduce input/netmetrics and refactor netflow input metrics {pull}38055[38055]
 - Update Salesforce module to use new Salesforce input. {pull}37509[37509]
+
 - Fix high IO and handling of a corrupted registry log file. {pull}35893[35893]
+- aws-s3 input - Document the transparent decompression of gzip files. {pull}39788[39788]
```

Comment:
Thanks for the update! I dont think we need a changelog here for documentation change.